### ISSUE TITLE
Versioning: pull and rollback buttons don't work

### ISSUE THREAD

**Issue by robin-monnier** [keymetrics/keymetrics-support#127]
Sometimes pull and rollback buttons don't work

**Comment by vmarchaud** [comment]
@rmonnier What is sometimes ? Are you sure it isn't just the pull that can't be upgraded ?

**Comment by martingsaunders** [comment]
I have had `File modified (unstaged changes)` as an error for ages and no Pull button. If I SSH in to the server though `git status` shows it as clean.

It used to work until I updated PM2 relatively recently. Then it would work intermittently, giving no console output in the popup window on Keymetrics until I did `pm2 update` on the server, but then it would stop working again soon after. Now it doesn't work at all.

I have deployed by both `pm2 deploy ...` on my local machine and `pm2 start ...` on the server via SSH so I wonder if the two routes are fighting each other maybe.
